modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,568 @@
+/*global jQuery _ wp*/
```

Comment:
I seem to recall someone remarking that underscore isn't available in Calypso -- I'm currently using `_.map()` `_.pluck()` `_.clone()` and `_.defaults()` -- are there recommended ways to abstract these out of either underscore or lodash or to make it work equally well with either?

Is `jQuery` available in Calypso?  I'm currently using `jQuery.closest()` and `jQuery.find()` to do some traversal, but can probably rewrite it in vanilla javascript if necessary.

Reply:
Lodash is used in Calypso, but typically with import syntax. (e.g. `import { pluck } from 'lodash' )
We've really tried hard not to use jQuery in Calypso at all, so I'd recommend against it if you can. If you're trying to inspect or manipulate the DOM using jQuery, you probably need to be careful doing this as it can really interfere with the way React works.

Reply:
Yeah, no underscore in Calypso, only lodash. They're sometimes compatible, but I wouldn't rely on it.

jQuery isn't available globally in Calypso. Gutenberg pulls in v3 for some things now, but they're working on removing it.